tests/bootstrap_project/test/project_test.dart

```diff
@@ -0,0 +1,432 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:test/test.dart';
+import 'package:path/path.dart' as path;
+
+void main() async {
+  Directory.current = path.join(Directory.current.path, '..', '..');
+  final rootPath = Directory.current.path;
+  final cliPath = path.join(rootPath, 'tools', 'serverpod_cli');
+  final tempPath = path.join(rootPath, 'temp');
+
+  final timeout = Timeout(Duration(minutes: 5));
+
+  setUpAll(() async {
+    await Process.run(
+      'dart',
+      ['pub', 'global', 'activate', '-s', 'path', '.'],
+      workingDirectory: cliPath,
+    );
+
+    await Process.run('mkdir', ['temp'], workingDirectory: rootPath);
+    Directory.current = tempPath;
+  });
+
+  group('Given a clean state', () {
+    final (projectName, commandRoot) = createRandomProjectName(tempPath);
+
+    late Process createProcess;
+
+    tearDown(() async {
+      createProcess.kill();
+
+      await Process.run(
+        'docker',
+        ['compose', 'down', '-v'],
+        workingDirectory: commandRoot,
+      );
+    });
+
+    test(
+        'when creating a new project then the project is created successfully and can be booted',
+        () async {
+      createProcess = await Process.start(
+        'serverpod',
+        ['create', projectName, '-v'],
+        workingDirectory: tempPath,
+        environment: {
+          'SERVERPOD_HOME': rootPath,
+        },
+      );
+
+      createProcess.stdout.transform(Utf8Decoder()).listen(print);
+      createProcess.stderr.transform(Utf8Decoder()).listen(print);
+
+      var createProjectExitCode = await createProcess.exitCode;
+      expect(createProjectExitCode, 0);
```

Comment:
Suggestion: add a reason so that we understand the failure in the CI messages.

```suggestion
      expect(createProjectExitCode, 0, reason: 'Should have successfully created project but exit code was != 0',);
```